electron-builder.json

```diff
@@ -3,6 +3,22 @@
   "directories": {
     "output": "app-builds"
   },
+    "files": [
```

Comment:
Hi @hervehobbes 

Could we just add `main.js` and `/dist` and ignore everything else?